lib/fog/compute/google/models/disk.rb

```diff
@@ -45,7 +45,7 @@ def destroy(async = true)
           requires :name, :zone
 
           data = service.delete_disk(name, zone_name)
-          operation = Fog::Compute::Google::Operations.new(:service => service).get(data.body["name"], data.body["zone"])
+          operation = Fog::Compute::Google::Operations.new(:service => service).get(data.name, data.zone)
```

Comment:
In other PRs you've been referencing these as symbols. Is there a difference here?